extensions/amazon-lambda/runtime/pom.xml

```diff
@@ -45,6 +49,7 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+
```

Comment:
How's that related? Did this really pass the automated formatter?

Reply:
It actually did... weird.

Reply:
Removed the extra line.